grails-app/services/org/pih/warehouse/inventory/StockMovementService.groovy

```diff
@@ -1890,20 +1889,14 @@ class StockMovementService {
                 }
 
                 if (!requisitionItem) {
-                    throw new IllegalArgumentException("Could not find stock movement item with ID ${stockMovementItem.id}")
+                    throw new IllegalArgumentException("Could not find requisition item for stock movement item with ID ${stockMovementItem.id}")
```

Comment:
This is unnecessary. A requisition item is *for all intents and purposes* a stock movement item. You cannot have a stock movement item without one.

Reply:
ok, good to know. Backed out in c4f461429